src/main/java/uk/gov/hmcts/darts/authentication/component/SoapRequestInterceptor.java

```diff
@@ -202,4 +218,33 @@ private void logPayloadMessage(String messagePrefix, WebServiceMessage message)
             log.error("Failed to write SOAP message", ex);
         }
     }
+
+    private void verifyBasicAuthorisationRequestIsAllowed(String userName, SaajSoapMessage message) {
+        if (isContextRegistryRequest(message)) {
+            return;
+        }
+
+        if (!securityProperties.getExternalServiceBasicAuthorisationWhitelist().contains(userName)) {
```

Comment:
Not essential but we could add a reusable function on the properties to determine if the user is white listed